test/terraform_aws_s3_private_bucket_test.go

```diff
@@ -250,3 +281,40 @@ func TestTerraformAwsS3PrivateBucketCustomPolicy(t *testing.T) {
 
 	aws.AssertS3BucketPolicyExists(t, awsRegion, testName)
 }
+
+func TestTerraformAwsS3PrivateBucketNoLoggingBucket(t *testing.T) {
+	t.Parallel()
+
+	tempTestFolder := test_structure.CopyTerraformFolderToTemp(t, "../", "examples/simple-no-logging")
+
+	// Give this S3 Bucket a unique ID for a name tag so we can distinguish it from any other Buckets provisioned
+	// in your AWS account
+	testName := fmt.Sprintf("terratest-aws-s3-private-bucket-no-logging-%s", strings.ToLower(random.UniqueId()))
+	loggingBucket := fmt.Sprintf("%s-no-logs", testName)
+	awsRegion := "us-west-2"
+
+	terraformOptions := &terraform.Options{
+		// The path to where our Terraform code is located
+		TerraformDir: tempTestFolder,
+
+		// Variables to pass to our Terraform code using -var options
+		Vars: map[string]interface{}{
+			"test_name":      testName,
+			"region":         awsRegion,
+			"logging_bucket": loggingBucket,
```

Comment:
The fact that this passes in the loggin_bucket and yet `AssertS3BucketLoggingNotEnabled` passes makes me think something is off with the either the test or the conditional logic around configuring the logging.

Reply:
Nevermind, see @dynamike's comments below. Momentarily forgot that `examples/simple-no-logging` was no longer using the logging_bucket.